docs/docs/components/quickstart/nodejs/actions/README.md

```diff
@@ -0,0 +1,455 @@
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Name",
+    }
+  },
+  async run() {
+    return `hello world!`
+  },
+}
+```
+
+Next, update the `run()` function to reference `this.name` in the return value.
+
+```javascript
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Name",
+    }
+  },
+  async run() {
+    return `hello ${this.name}!`
+  },
+}
+```
+
+Finally, update the component version to `0.0.2`. If you fail to update the version, the CLI will throw an error.
+
+```javascript
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.2",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Name",
+    }
+  },
+  async run() {
+    return `hello ${this.name}!`
+  },
+}
+```
+
+Save the file and run the `pd publish` command again to update the action in your account.
+
+```
+pd publish action.js
+```
+
+The CLI will update the component in your account with the key `action_demo`. You should see something like this:
+
+```
+sc_Egip04  Action Demo                             0.0.2    just now             action_demo
+```
+
+Next, let's update the action in the workflow from the previous example and run it.
+
+1. Hover over the action in your workflow — you should see an update icon at the top right. Click the icon to update the action to the latest version and then save the workflow. If you don't see the icon, verify that the CLI successfully published the update.
+
+   ![image-20210411164514490](https://res.cloudinary.com/pipedreamin/image/upload/v1618550730/docs/components/image-20210411164514490_qghbzf.png)
+
+2. After saving the workflow, you should see an input field appear. Enter a value for the `Name` input (e.g., `foo`). 
+   ![image-20210411165053922](https://res.cloudinary.com/pipedreamin/image/upload/v1618550730/docs/components/image-20210411165053922_pckn5y.png)
+3. Deploy the workflow and click **RUN NOW**
+
+You should see `hello foo!` (or the value you entered for `Name`) as the value returned by the step.
+
+### Use an npm Package
+
+Next, we'll update the component to get data from the Star Wars API using the `axios` npm package. To use the `axios` package, just require it.
+
+```javascript
+const axios = require("axios")
+
+module.exports = {
+  name: "Action Demo",
+  description: "This is a demo action",
+  key: "action_demo",
+  version: "0.0.2",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Name",
+    }
+  },
+  async run() {
+    return `hello ${this.name}!`
+  },
+}
+```
+
+> **Note:** To use most npm packages on Pipedream, just require them — there is no `package.json` or `npm install` required.
```

Comment:
In the docs I normally wrap require with backticks when used in this context to note that it's code.